src/test/java/com/limechain/chain/StateRootHashesIntegrationTest.java

```diff
@@ -0,0 +1,40 @@
+package com.limechain.chain;
+
+import com.limechain.chain.spec.ChainSpec;
+import com.limechain.runtime.StateVersion;
+import com.limechain.trie.TrieStructureFactory;
+import org.apache.tomcat.util.buf.HexUtils;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.util.Map;
+import java.util.Objects;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class StateRootHashesIntegrationTest {
+
+    // TODO: Refactor these hardcoded paths after we refactor to extract a logical ChainSpec module
```

Comment:
It is actually not uncommon for tests to have hardcoded setups like this one. Given that it doesn't take that much space this TODO could be omitted.

Reply:
The concern you're raising has a way bigger scope than just this test so the TODO would be better positioned more outwardly. What I mean is that maybe it's not a bad idea for us to have a TODO.md where have improvement suggestions such as this one that's maybe in root folder so it's more easily viewable by all. 

Anyway this would require hearing the other team members opinions so it's a discussion item for tomorrow given that we are past working hours.